storage/oasis/nodeapi/history/sdk.go

```diff
@@ -0,0 +1,35 @@
+package history
+
+import (
+	"context"
+	"fmt"
+
+	sdkConfig "github.com/oasisprotocol/oasis-sdk/client-sdk/go/config"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+
+	"github.com/oasisprotocol/oasis-indexer/config"
+)
+
+// SDKConnect creates an Oasis SDK Connection to a node running the current
+// version of oasis-node.
+func SDKConnect(ctx context.Context, chainContext string, nodeConfig *config.NodeConfig, fastStartup bool) (connection.Connection, error) {
```

Comment:
Nit, nonblocking: Not sure this has to do with history much ... Can you think of a better home? `storage/oasis`? Same for raw_grpc.go.

Reply:
oh ok I can move these into a separate directory :+1: